cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -14,16 +14,19 @@
  * the License.
  */
 
-function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_ACTIONS, MyCDAPDataSource, $sce, myCdapUrl) {
+function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_ACTIONS, MyCDAPDataSource, $sce, myCdapUrl, $timeout) {
   'ngInject';
 
   var dataSrc = new MyCDAPDataSource($scope);
   var pollPromise;
   //Collapsing LogViewer Table Columns
   var columnsList = [];
   var collapseCount = 0;
+  //If we are replacing a previously generated file we need to manuually revoke the object URL to avoid memory leaks
+  this.textFile = null;
```

Comment:
I don't think we need this here. We are calling `setDefault()` at line 69 and that should set `this.textFile = null`